source/public/mvp-1/probate.slim

```diff
@@ -8,7 +8,7 @@ form
     .small-12.columns
       br
       h4 Are you paying a fee for a probate case?
-      p These cases are usually about the property and belongings of someone who has died.
+      p Probate cases are about the property and belongings of someone who has died.
```

Comment:
It says 'These cases' on the paper form, but no problem if you want it to be different online